internal/cmd/commands.go

```diff
@@ -57,1268 +62,1283 @@ func initCommands(ui, serverCmdUi cli.Ui, runOpts *RunOptions) {
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "rdp",
 			}, nil
 		},
 		"connect ssh": func() (cli.Command, error) {
 			return &connect.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "ssh",
 			}, nil
 		},
 
 		"database": func() (cli.Command, error) {
 			return &database.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 			}, nil
 		},
 		"database init": func() (cli.Command, error) {
 			return &database.InitCommand{
-				Server: base.NewServer(base.NewCommand(ui)),
+				Server: base.NewServer(base.NewCommand(ui, opts...)),
 			}, nil
 		},
 		"database migrate": func() (cli.Command, error) {
 			return &database.MigrateCommand{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 			}, nil
 		},
 
 		"credential-libraries": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 			}, nil
 		},
 		"credential-libraries read": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "read",
 			}, nil
 		},
 		"credential-libraries delete": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "delete",
 			}, nil
 		},
 		"credential-libraries list": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "list",
 			}, nil
 		},
 		"credential-libraries create": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "create",
 			}, nil
 		},
 		"credential-libraries create vault": func() (cli.Command, error) {
 			return &credentiallibrariescmd.VaultCommand{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "create",
 			}, nil
 		},
 		"credential-libraries create vault-generic": func() (cli.Command, error) {
 			return &credentiallibrariescmd.VaultGenericCommand{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "create",
 			}, nil
 		},
 		"credential-libraries create vault-ssh-certificate": func() (cli.Command, error) {
 			return &credentiallibrariescmd.VaultSshCertificateCommand{
-				Command: base.NewCommand(ui),
+				Command: base.NewCommand(ui, opts...),
 				Func:    "create",
 			}, nil
 		},
 		"credential-libraries update": func() (cli.Command, error) {
 			return &credentiallibrariescmd.Command{
-				Command: base.NewCommand(ui),
-				Func:    "update",
-			}, nil
-		},
-		"credential-libraries update vault": func() (cli.Command, error) {
```

Comment:
This looks like a breaking change, have we waited long enough to remove this?

Reply:
I will double check but I'm pretty sure I checked this and the answer is yes

Reply:
See updated changelog